core/internal/cltest/factories.go

```diff
@@ -732,7 +732,16 @@ func MustInsertExternalInitiatorWithOpts(t *testing.T, db *gorm.DB, opts Externa
 	ei.OutgoingToken = opts.OutgoingToken
 	token := auth.NewToken()
 	ei.AccessKey = token.AccessKey
-	err := db.Create(&ei).Error
+	ei.Salt = utils.NewSecret(utils.DefaultSecretSize)
+	hashedSecret, err := auth.HashedSecret(token, ei.Salt)
+	ei.HashedSecret = hashedSecret
+	require.NoError(t, err)
+	sql := `INSERT INTO external_initiators (created_at, updated_at, name, url, access_key, salt, hashed_secret, outgoing_secret, outgoing_token)
```

Comment:
Could you extract a method for this to bridges/orm

Reply:
@RensR does it need to be done even for tests? (I don't know for sure, therefore double checking)

Reply:
It seems it already exists CreateExternalInitiator(externalInitiator *ExternalInitiator) so you could use the existing one

Reply:
Done